src/main/resources/features/moth/54 - Print MOT certificate.feature

```diff
@@ -0,0 +1,23 @@
+@mothprint @mothint
+Feature: 54 - Check whether MOT certificates can be downloaded after entering the V5C number
+
+  Scenario: A MOTH user searches for a vehicle with an MOT test and downloads the MOT certificate
+    Given I browse to /
+    And I enter "CGSENOK" in the registration field
+    When I press the "Continue" button
+
+    And I click the accordion section with the id "mot-history-description"
+    And I click the first "View test certificate" text
+    And The page contains "Enter latest V5C number"
+
+    And I enter "00000000000" in the field with id "v5c-print-cert-input"
+    And I press the first "Show test certificate" button
+    And The page contains "Check that the V5C number you entered is correct"
+
+    And I enter "12312312312" in the field with id "v5c-print-cert-input"
+    And I press the first "Show test certificate" button
+
+    And The page contains "View certificate"
+    And I check the "Return to MOT test history" button is enabled
+    And I click the button with id "cert-download-link"
+
```

Comment:
Can include a check if the PDF of the certificate looks correct here

Reply:
Checks added